recipes/libvpx/all/conanfile.py

```diff
@@ -160,7 +160,7 @@ def _execute_configure(self):
         elif is_msvc(self):
             vc_version = str(self.settings.compiler.version)
             if Version(self.settings.compiler.version) > "100": # New msvc compiler to triplet used in configure script for vpx
-                vc_version = {"190": "14", "191": "15", "192": "16"}[vc_version]
+                vc_version = {"190": "14", "191": "15", "192": "16", "193": "17"}[vc_version]
```

Comment:
```suggestion
                vc_version = {"170": "11", "180": "12", "190": "14", "191": "15", "192": "16", "193": "17"}[vc_version]
```

Reply:
Could you also remove this ConanInvalidConfiguration you have added in last PR at the end of this branching? ConanInvalidConfiguration should only be called in validate() and there is already a logic in validate() checking that we cannot use other compilers not managed be current logic in `_execute_configure()`.

Reply:
Ok good point :)